src/components/ReportCategory/ReportCategory.test.tsx

```diff
@@ -9,13 +9,13 @@ import {ReportCategory} from './ReportCategory';
 describe('ReportCategory', () => {
 
```

Comment:
Great job with this test, I can see here that the use of a `before each` could've been a good addition. It would be cleaner in my opinion to have the component to render only once instead of for every `it block`.
Thanks for the hard work